tasks/tensorflow.py

```diff
@@ -0,0 +1,68 @@
+from os import cpu_count
+
+from os.path import exists, join
+
+from shutil import rmtree
+
+from subprocess import call, check_output
+
+from invoke import task
+
+from faasmtools.env import (
+    THIRD_PARTY_DIR,
+)
+
+from faasmtools.build import (
+    BASE_CONFIG_CMD,
+    WASM_CFLAGS,
+    WASM_CXXFLAGS,
+    WASM_HOST,
+    WASM_LDFLAGS,
+)
+
+
+@task
+def lite(ctx, clean=False):
+    """
+    Compile and install Tensorflow Lite lib
+    """
+    tf_dir = join(THIRD_PARTY_DIR, "tensorflow")
+    tf_lite_dir = join(tf_dir, "tensorflow", "lite")
+    tf_make_dir = join(tf_lite_dir, "tools", "make")
+
+    download_check_dir = join(tf_make_dir, "downloads", "absl")
+    if not exists(download_check_dir):
+        download_script = join(tf_make_dir, "download_dependencies.sh")
+        check_output(download_script, shell=True)
+
+    cores = cpu_count()
+    make_cores = int(cores) - 1
+
+    make_target = "lib"
+    make_cmd = ["make -j {}".format(make_cores)]
+    make_cmd.extend(BASE_CONFIG_CMD)
+    make_cmd.extend(
+        [
+            'CFLAGS="{} -ftls-model=local-exec"'.format(" ".join(WASM_CFLAGS)),
+            'CXXFLAGS="{}"'.format(" ".join(WASM_CXXFLAGS)),
+            'LDFLAGS="{} -Xlinker --max-memory=4294967296"'.format(
+                " ".join(WASM_LDFLAGS)
+            ),
+            "MINIMAL_SRCS=",
+            "TARGET={}".format(WASM_HOST),
+            "BUILD_WITH_MMAP=false",
+            'LIBS="-lstdc++"',
+            '-C "{}"'.format(tf_dir),
+            "-f tensorflow/lite/tools/make/Makefile",
+        ]
+    )
+
+    make_cmd.append(make_target)
+
+    clean_dir = join(tf_make_dir, "gen", "wasm32-unknown-wasi_x86_64")
+    if clean and exists(clean_dir):
+        rmtree(clean_dir)
+
+    res = call(" ".join(make_cmd), shell=True, cwd=tf_lite_dir)
```

Comment:
You probably want to add a step to install the library in the right library path which is `/usr/local/faasm/llvm-sysroot/lib/wasm32-wasi/<lib>.a`, this directory is declared as a variable in `faasmtools/build.py`.